### ISSUE TITLE
Superior cerebellar dysplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4930]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0007033

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
-- ARMC9, Joubert 30

**Your nano-attribution (ORCID)**
